queries.sql

```diff
@@ -55,3 +55,107 @@ SELECT
     weight_kg
 FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+
+SELECT
+    id,
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg,
+    species
+FROM animals;
```

Comment:
- [OPTIONAL] Verifying the changes is very good. However, it will be nice to only verify a specific column that was changed.

See example:
```sql
 SELECT species from animals;
```